### DISCUSSION TITLE
update several records via filters

### DISCUSSION THREAD

**Discussion by moviex1** [Laravel-Backpack/community-forum#1244] [Q&A (Help)]
Is it possible to update several records in admin panel, by some criteria. like if user has specific role and we want to change some fields or relations.

[a comment by an automated account is not shown]

**Comment by jcastroa87** [comment]
Hello @moviex1 

Could you provide more details about what you’d like to accomplish? For example:

- What specific criteria do you want to use for selecting the records (e.g., roles, permissions, other conditions)?
- What fields or relations are you looking to update?
- Are you thinking of a bulk operation within the admin panel, or do you have something else in mind?

Understanding your needs more clearly will help in suggesting the best solution. Let me know!

If you want to filter results on datatable and be able to update right there, you want to try our Premium Add-on [Editable Columns](https://backpackforlaravel.com/products/editable-columns)

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity, I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
